### DISCUSSION TITLE
Copy from markdown preview

### DISCUSSION THREAD

**Discussion by jbobko** [zed-industries/zed#13395] [Feature Requests]
Is there a way to copy text from the Markdown preview? 

Currently, I need to find the section in the raw markdown file to copy it.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
